packages/msw-addon/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "noImplicitAny": true,
+    "module": "CommonJS",
+    "target": "es5",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
+    "declaration": true
+  },
+  "include": ["src"],
+  "exclude": ["node_modules"]
```

Comment:
I generally advise to always blacklist `**/*.test.*` files. Just so we don't have to worry in case we decide to keep test files closer to the source code. Those test files mustn't be transpiled.

Reply:
Agreed!

Reply:
Fixed!